protobuf/.gitignore

```diff
@@ -0,0 +1,5 @@
+# Ignore Gradle project-specific cache directory
+.gradle
```

Comment:
this file should be present only in root module

Reply:
ok